docs/index.html

```diff
@@ -35,390 +36,292 @@
-                "ring": HOLD,
+                "ring": MAINTAIN,
                 "label": "Ionic",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": ADOPT,
+                "ring": CORE,
                 "label": "SCSS",
                 "active": true,
-                "moved": 0
+                "moved": -1
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": ADOPT,
+                "ring": CORE,
                 "label": "TypeScript",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": ADOPT,
+                "ring": COMPETENT,
                 "label": "Swift",
                 "active": true,
-                "moved": 0
+                "moved": -1
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": TRIAL,
+                "ring": COMPETENT,
                 "label": "Kotlin",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": TRIAL,
+                "ring": COMPETENT,
                 "label": "Dart",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": PLATFORM,
-                "ring": TRIAL,
+                "ring": COMPETENT,
                 "label": "Flutter Mobile",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": ASSESS,
+                "ring": EXPLORE,
                 "label": "NodeJS",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": PLATFORM,
-                "ring": ADOPT,
+                "ring": CORE,
                 "label": "API Platform",
                 "active": true,
                 "moved": 0
             },
             {
                 "quadrant": LANGUAGE,
-                "ring": ASSESS,
+                "ring": EXPLORE,
                 "label": "GraphQL",
                 "active": true,
                 "moved": 0
             },
-            {
-                "quadrant": LANGUAGE,
-                "ring": ADOPT,
-                "label": "OpenAPI (Swagger)",
-                "active": true,
-                "moved": 0
-            },
-            {
-                "quadrant": LANGUAGE,
-                "ring": HOLD,
-                "label": "SOAP",
-                "active": true,
-                "moved": 0
-            },
+            // {
+            //     "quadrant": LANGUAGE,
+            //     "ring": COMPETENT,
+            //     "label": "OpenAPI (Swagger)",
+            //     "active": true,
+            //     "moved": -1
+            // },
```

Comment:
Why was this removed? Because we still use it quite a lot.

Reply:
I removed this because is so standard that is doesn't set us apart in any way